changelogs/index.html

```diff
@@ -0,0 +1,121 @@
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <style>
+        .bd-placeholder-img {
+            font-size: 1.125rem;
+            text-anchor: middle;
+            -webkit-user-select: none;
+            -moz-user-select: none;
+            -ms-user-select: none;
+            user-select: none;
+        }
+
+        @media (min-width: 768px) {
+            .bd-placeholder-img-lg {
+                font-size: 3.5rem;
+            }
+        }
+    </style>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <a class="navbar-brand" href="#">5zig Reborn</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+                aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" href="/">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/downloads/">Downloads</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://secure.5zigreborn.eu/panel">Panel</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://status.5zigreborn.eu">Status</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/faq">FAQ</a>
+                    </li>
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#">Changelogs <span class="sr-only">(current)</span></a>
+                    </li>
+                </ul>
+                <span class="navbar-text">
+                    <a href="https://www.patreon.com/bePatron?u=21831307" style="height:37px"
+                        data-patreon-widget-type="become-patron-button">Become a
+                        Patron!</a>
+                    <script async src="https://c6.patreon.com/becomePatronButton.bundle.js"></script>
+                </span>
+            </div>
+        </nav>
+    </header>
+
+    <main role="main">
+
+        <section class="jumbotron text-center">
+            <div class="container">
+                <h1 class="jumbotron-heading">The 5zig Mod - Changelogs</h1>
+                <p class="lead text-muted">All-in-one modification for Minecraft 1.8.9-1.15.2. <br>
+                    Revamp the way you play the game!</p>
+            </div>
+        </section>
+
+        <section class="text-center">
+            <h3>Changelogs</h3>
```

Comment:
I'd change this to "Stable Changelogs". I'll eventually add a section for beta changelog with commit messages pulled from GitHub.

Reply:
Changed to `Changelog (Stable)`